google-cloud-clients/google-cloud-firestore/src/main/java/com/google/cloud/firestore/Firestore.java

```diff
@@ -131,4 +133,7 @@
    */
   @Override
   void close() throws Exception;
+
+  /** Gets a FirestoreRpc. */
+  FirestoreRpc getFirestoreRpc();
 }
```

Comment:
I think the location of the getter should be placed above close()

Reply:
Done